src/main/webapp/resources/user/mypage.css

```
@@ -1,5 +1,5 @@
-body { font-family: 'Malgun Gothic', sans-serif; background-color: #f0f2f5; display: flex; justify-content: center; align-items: center; min-height: 100vh; margin: 0; }
-.mypage-container { background: white; padding: 40px; border-radius: 12px; shadow: 0 8px 20px rgba(0,0,0,0.1); width: 450px; box-shadow: 0 4px 15px rgba(0,0,0,0.1); }
+body { font-family: 'Malgun Gothic', sans-serif; display: flex; flex-direction: column; align-items: stretch; min-height: 100vh; background-color: #f0f2f5; margin: 0; }
+.mypage-container {  margin: 40px auto 80px; background: white; padding: 40px; border-radius: 12px; shadow: 0 8px 20px rgba(0,0,0,0.1); width: 450px; box-shadow: 0 4px 15px rgba(0,0,0,0.1); }
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)